graylog2-web-interface/src/components/events/events/hooks/useEventAction.tsx

```diff
@@ -19,16 +19,18 @@ import React, { useMemo } from 'react';
 import usePluggableEventActions from 'components/events/events/hooks/usePluggableEventActions';
 import { MenuItem } from 'components/bootstrap';
 import LinkToReplaySearch from 'components/event-definitions/replay-search/LinkToReplaySearch';
+import useSendEventActionTelemetry from 'components/events/events/hooks/useSendEventActionTelemetry';
 
 const useEventAction = (event) => {
```

Comment:
While we are on it, can we type the `event` parameter here, so it is not implicitly `any`?